src/install-pnpm/run.ts

```diff
@@ -2,19 +2,21 @@ import { addPath, exportVariable } from '@actions/core'
 import fetch from '@pnpm/fetch'
 import { spawn } from 'child_process'
 import { remove, ensureFile, writeFile, readFile } from 'fs-extra'
-import path from 'path'
+import { join } from 'path'
```

Comment:
Is there a reason for this change? I actually prefer qualified `path.join()` over ambiguous `join()`. And since `path` is a built-in Node.js module, importing the whole thing is not going to increase the bundle size either.